status.go

```diff
@@ -15,7 +15,8 @@ const (
 
 	// timeFormatShort is the package time format of date-only timestamps
 	// from a NIS.
-	timeFormatShort = "2006-01-02"
+	timeFormatShort    = "2006-01-02"
+	timeFormatShortest = "01/02/06"
```

Comment:
I suspect you'll always want to look for both of these for short format, so let's do:

```go
// timeFormatsShort contains package time formats for date-only timestamps.
var timeFormatsShort = []string{"2006-01-02", "01/02/06"}
```